addons/web/tooling/_eslintignore

```diff
@@ -370,3 +370,6 @@ addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js
 # Whitelist the shop floor module
 !mrp_workorder
 !mrp_workorder/**
+
+!ai
```

Comment:
Unrelaterd, is it ?

Reply:
Nevermind, did not see it was related to master AI. PR name could have been a bit more precise ^^ .